src/dune_rules/cram/cram_exec.ml

```diff
@@ -100,7 +103,8 @@ let translate_path_for_sh =
     match cygpath with
     | None -> User_error.raise [ Pp.text "Unable to find cygpath in PATH" ]
     | Some cygpath ->
-      Process.run_capture_line Strict cygpath [ Path.to_absolute_filename fn ]
+      Process.run_capture_line ~display:Dune_engine.Display.quiet Strict cygpath
```

Comment:
How did you decide to use `quiet` here but `!Clflags.display` above? Seems pretty arbitrary.

Reply:
(Same question in a few other places.)